src/bio/bio_monitor.c

```diff
@@ -623,6 +627,35 @@ bio_init_health_monitoring(struct bio_blobstore *bb, char *bdev_name)
 	D_ASSERT(channel != NULL);
 	bb->bb_dev_health.bdh_io_channel = channel;
 
+	/** register DAOS metrics to export NVMe stats */
+	D_ALLOC_PTR(binfo);
+	if (binfo == NULL) {
+		D_WARN("Failed to allocate binfo\n");
+		return 0;
+	}
+#define X(field, fname, desc, unit, type)				\
```

Comment:
(style)  Macros with multiple statements should be enclosed in a do - while loop